### PR TITLE
Backport "Fixes for sos-report >= 4.0 versions" to 0.69

### PR DESCRIPTION
**Pull request by ndokos**
Backport of #2372.

### CODE REVIEW

**Comment by ndokos** [comment]
Tested on F33 (sos v4.0) and F34 (sos v4.1)

File: agent/util-scripts/pbench-sysinfo-dump
Comment (webbnh) [inline agent/util-scripts/pbench-sysinfo-dump:134]: While this obviously works, it's more complex than it needs to be; e.g.:
```
	printf -- "${_sos_report_cmd}" > ${_cmd}
```
If you're worried about there being percent signs in the command string, then use something even simpler, like:
```
	echo ${_sos_report_cmd} > ${_cmd}
```

**Review by webbnh — APPROVED** [review]
It looks good except for line 144 (and the nit I called out below).

**Review by dbutenhof — APPROVED** [review]

**Comment by portante** [comment]
Before we consider this back-port let's get the original changes merged into `main` and then revisit.

**Review by portante — CHANGES_REQUESTED** [review]
Let's hold off on this work until we get PR #2372 merged into `main`.

File: agent/util-scripts/pbench-sysinfo-dump
Comment (ndokos) [reply agent/util-scripts/pbench-sysinfo-dump:134]: See my reply in #2372.

**Comment by ndokos** [comment]
Changed the comment on l.144

**Review by portante — APPROVED** [review]

File: agent/util-scripts/pbench-sysinfo-dump
Comment (webbnh) [inline agent/util-scripts/pbench-sysinfo-dump:144]: If we want this to be perfect, then

s/Version/Versions/

but I'm not going to quibble.

**Review by webbnh — APPROVED** [review]

File: agent/util-scripts/pbench-sysinfo-dump
Comment (ndokos) [reply agent/util-scripts/pbench-sysinfo-dump:144]: Sigh ... I'll leave it as is for now.

**Pull request merged**
